Source/SwiftLintFramework/Rules/RuleConfigurations/ImplicitlyUnwrappedOptionalConfiguration.swift

```diff
@@ -13,17 +13,17 @@ public enum ImplicitlyUnwrappedOptionalModeConfiguration: String {
     }
 }
 
-public struct ImplicitlyUnwrappedOptionalConfiguration: RuleConfiguration, Equatable {
-    private(set) var severity: SeverityConfiguration
-    private(set) var mode: ImplicitlyUnwrappedOptionalModeConfiguration
+public struct ImplicitlyUnwrappedOptionalConfiguration: RuleConfiguration, Equatable, SeverityBasedRuleConfiguration {
```

Comment:
```suggestion
public struct ImplicitlyUnwrappedOptionalConfiguration: SeverityBasedRuleConfiguration, Equatable {
```
Or would you really like to keep both? In this case, `SeverityBasedRuleConfiguration` could be renamed into `SeverityConfigurationProvider` or something similar.